uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/controllers/sub-controllers/toil-request.controller.js

```diff
@@ -57,43 +47,52 @@ define([
     function calculateToilExpiryDate () {
       // blocks the expiry date from updating if this is an existing request
       // and user is not a manager or admin
-      if (!vm.canManage && vm.request.id) {
-        return $q.resolve(vm.request.toil_expiry_date);
+      if (!parentCtrl.canManage && parentCtrl.request.id) {
+        return $q.resolve(parentCtrl.request.toil_expiry_date);
       }
 
       // skips calculation of expiration date if request never expires
       // according to admin setting
-      if (!vm.requestCanExpire) {
-        vm.request.toil_expiry_date = false;
+      if (!parentCtrl.requestCanExpire) {
+        parentCtrl.request.toil_expiry_date = false;
         return $q.resolve(false);
       }
 
       return getReferenceDate().catch(function (errors) {
-        if (errors.length) vm.errors = errors;
+        if (errors.length) parentCtrl.errors = errors;
```

Comment:
again, please *never* inline your code

Reply:
I ***never*** do that since I am a huge enemy of inline code. In this case I used auto-replacement that's why I didn't notice it.

This is what I prefer `if (detailsController.isMode('view')) { return; }`